README.md

```diff
@@ -1 +1,202 @@
-# math-magician-2
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+    <img src="./src/assets/media/Math magicians-icon.png" alt="Math magicians" width="150"  height="auto" />
```

Comment:
- [ ] Please either remove the logo or add the correct `path` for your logo because your image link is broken and that's the reason why your `README` logo is not visible, which looks unprofessional.

![Screenshot from 2023-09-17 02-14-22](https://github.com/EvansSnave/math-magician-2/assets/66476329/d0fce533-9b8b-4b58-832e-b2ffe3f2ccd8)